tests/test_http_crawler.py

```diff
@@ -159,3 +176,22 @@ def test_extract_urls_from_css():
         '/assets/somefont.eot',
         '/assets/somefont.ttf',
     }
+
+
+def test_ssl_verify_false():
+    serve()
+
+    rsps = list(http_crawler.crawl('https://localhost:8002', verify=False))
+
+    assert len(rsps) == 1
+
+
+def test_ssl_verify_true():
+    serve()
+
+    e = None
+    try:
+        list(http_crawler.crawl('https://localhost:8002', verify=True))
+    except SSLError:
+        e = True
+    assert e
```

Comment:
You should use [`pytest.raises`](http://pytest.readthedocs.io/en/reorganize-docs/new-docs/user/pytest_raises.html) here.